configs.js

```diff
@@ -222,6 +222,23 @@ function loadStandaloneConfig() {
 
         metrics = conf.metrics;
 
+        if (_.isArray(configs.ignore)) {
+            let filtered = metrics;
```

Comment:
Why don't we do something like:

```
metrics = metrics.filter((metric) => {
    return !configs.ignore.includes(current.info().name);
})
```

Reply:
You cannot use info directly, must pass 3 args to the metric construct

Reply:
Why do we need to use the metric constructor?

Reply:
yea, in order to access the info method